sig-aws/charter.md

```diff
@@ -0,0 +1,55 @@
+
+### In scope
+
+Link to SIG [subprojects](https://github.com/kubernetes/community/tree/master/sig-aws#subprojects)
+
+#### Code, Binaries and Services
+
+Kubernetes integrations specific to AWS including:
+- Integrations, interfaces, libraries and extension points for all AWS services such as IAM, storage, networking, loadbalancers, registry, security, monitoring/logging at the instance or container level
+- Tools for Kubernetes APIs to work with AWS services including Amazon EKS
+- Prow, testgrid, perf dashboard integrations to expand and maintain testing (e2e, jobs) and scale-testing (load, density)
+- Support users on their issues and feature requests
+- Documentation for all things Kubernetes on AWS
+
+#### Cross-cutting and Externally Facing Processes
+
+- Consult with other SIGs and the community on how to apply mechanisms owned by SIG
+  AWS. Examples include:
+    - Review escalation implications of feature and API designs as it relates to core Kubernetes components (etcd, kubelet, apiserver, controller manager, scheduler)
+    - CSI, CNI, CRI implementation and design
+    - Cloud provider implementation and design
+    - Best practices for hardening add-ons or other external integrations such as KMS, LB, others.
+    - Implementing and hardening tests, scale tests and documentation
+
+### Out of scope
+
+SIG AWS is not for discussing bugs or feature requests outside the scope of Kubernetes. For example, SIG AWS should not be used to discuss or resolve support requests related to AWS Services. It should also not be used to discuss topics that other, more specialized SIGs own (to avoid overlap). Examples of such scenarios include:
+- Specification of CSI, CRI interfaces, cloudprovider binary (prefer: sig-storage, sig-node and sig-cloudprovider)
+- Container runtime (prefer: sig-node and sig-networking)
+- Resource quota (prefer: sig-scheduling)
+- Resource availability (prefer: sig-apimachinery, sig-network, sig-node)
+- Detailed design and scope of tests or tooling to run tests (prefer: sig-testing)
+- Detailed design and scope of scale tests or tooling to run scale tests (prefer: sig-scalability)
+- Reporting specific vulnerabilities in Kubernetes. Please report using these instructions: https://kubernetes.io/security/
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Subproject Creation
+
+SIG AWS delegates subproject approval to Technical Leads. See [Subproject creation - Option 1].
```

Comment:
Note that at present you have no Tech Leads spelled out in sigs.yaml, only Chairs.  Many sigs opt to say that Chairs also fulfill the duties of Tech Leads (or vice-versa) to use this option.

Reply:
Updating. Thanks!